src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/model/ExternalMovement.kt

```diff
@@ -1,14 +1,19 @@
 package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.model
 
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
 import java.time.LocalDate
 import java.time.LocalTime
-
+@Entity
+@Table(name = "movement_movements", schema = "domain")
 data class ExternalMovement(
-  val prisonNumber: String,
+  @Id val id: Long,
+  val prisoner: Long,
   val date: LocalDate,
   val time: LocalTime,
-  val from: String,
-  val to: String,
+  val origin: String,
+  val destination: String,
```

Comment:
Does this mean the API contract has changed? As that would need changes in the UI.

Reply:
I was thinking this might be better to change as the terminology we have in the data is origin and destination.